src/applications/mhv-landing-page/utilities/data/index.js

```diff
@@ -147,6 +147,18 @@ const resolveLandingPageLinks = (
       ]
   ).filter(isLinkData);
 
+  const medicalRecordsLinks = [
+    HEALTH_TOOL_LINKS.MEDICAL_RECORDS[0],
+    featureToggles[
+      FEATURE_FLAG_NAMES.mhvLandingPageShowShareMyHealthDataLink
+    ] && {
+      href: 'https://veteran.apps.va.gov/smhdWeb',
+      text:
+        'Share your personal health data on the Share My Health Data website',
+      isExternal: true,
```

Comment:
Per discussion in slack and the ticket, I think we want this to not be an external link:

```suggestion
```

Reply:
Re-confirmed that this is an external link -  it's styled as an external link and open in a new tab when clicked - but without displaying the text “(Opens in new tab)”